.pre-commit-config.yaml

```
@@ -1,11 +1,11 @@
 ---
 repos:
-  - repo: git://github.com/Lucas-C/pre-commit-hooks
+  - repo: https//github.com/Lucas-C/pre-commit-hooks
```

`:` is missing , please fix it 
```suggestion
  - repo: https://github.com/Lucas-C/pre-commit-hooks
```